source/plot/LinePlotData2D.ooc

```diff
@@ -24,14 +24,14 @@ LinePlotData2D: class extends PlotData2D {
 		super()
 		this lineStyle = lineStyle
 	}
-	init: func ~dataSeries (dataSeries: VectorList<FloatPoint2D>, label := "", colorRgba := ColorRgba new(), lineStyle := LineStyle Solid) {
+	init: func ~dataSeries (dataSeries: VectorList<FloatPoint2D>, label := "", colorRgba := ColorRgba transparent, lineStyle := LineStyle Solid) {
```

Comment:
By changing the none color to black, I did not have to change the output file since the color never made it to the output file.